runtime/lua/vim/lsp/rpc.lua

```diff
@@ -325,7 +323,15 @@ local function start(cmd, cmd_args, dispatchers, extra_spawn_params)
     end
     handle, pid = uv.spawn(cmd, spawn_params, onexit)
     if handle == nil then
-      error(string.format("start `%s` failed: %s", cmd, pid))
+      local msg = string.format("Spawning language server with cmd: `%s` failed", cmd)
+      if string.match(pid, "ENOENT") then
+        msg = msg .. ". The language server is either not installed, missing from PATH, or not executable."
+      else
+        msg = msg .. string.format(" with error message: %s", pid)
+      end
+      local _ = log.error() and log.error(msg)
+      vim.notify(msg)
```

Comment:
```suggestion
      vim.notify(msg, vim.log.levels.ERROR)
```

Also not sure if we need both log and notify.